WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -384,25 +385,42 @@ public static void openEditorForReblog(
     }
 
     public static void viewStatsInNewStack(Context context, SiteModel site) {
+        viewStatsInNewStack(context, site, null);
+    }
+
+    public static void viewStatsInNewStack(Context context, SiteModel site,  @Nullable StatsTimeframe statsTimeframe) {
```

Comment:
This should fix the checkstyle error.

```suggestion
    public static void viewStatsInNewStack(Context context, SiteModel site, @Nullable StatsTimeframe statsTimeframe) {
```